scripts/process_data.py

```diff
@@ -971,9 +1045,128 @@ def main(self) -> None:
+
+    def main(self) -> None:
+        """Process images into a nerfstudio dataset."""
+
+        self.output_dir.mkdir(parents=True, exist_ok=True)
+        image_dir = self.output_dir / "images"
+        image_dir.mkdir(parents=True, exist_ok=True)
+
+        summary_log = []
+
+        if self.data.suffix == ".zip":
+            with zipfile.ZipFile(self.data, "r") as zip_ref:
+                zip_ref.extractall(self.output_dir)
+                extracted_folder = zip_ref.namelist()[0].split("/")[0]
+            self.data = self.output_dir / extracted_folder
+            print(self.data)
+
+        if self.use_uncorrected_images:
+            polycam_image_dir = self.data / "keyframes" / "images"
+            polycam_cameras_dir = self.data / "keyframes" / "cameras"
+        elif (self.data / "keyframes" / "corrected_images").exists():
+            polycam_image_dir = self.data / "keyframes" / "corrected_images"
+            polycam_cameras_dir = self.data / "keyframes" / "corrected_cameras"
+        else:
+            polycam_image_dir = self.data / "keyframes" / "images"
+            polycam_cameras_dir = self.data / "keyframes" / "cameras"
+            self.crop_border_pixels = 0
+            CONSOLE.log("[bold yellow]Corrected images not found, using raw images. No border cropping will be done.")
+
+        if not polycam_image_dir.exists():
+            raise ValueError(f"Image directory {polycam_image_dir} doesn't exist")
+
+        # Copy images to output directory
+
+        polycam_image_filenames = []
+        for f in polycam_image_dir.iterdir():
+            polycam_image_filenames.append(f)
+        polycam_image_filenames = sorted(polycam_image_filenames, key=lambda fn: int(fn.stem))
+        num_images = len(polycam_image_filenames)
+        idx = np.arange(num_images)
+        if self.max_dataset_size != -1 and num_images > self.max_dataset_size:
+            idx = np.round(np.linspace(0, num_images - 1, self.max_dataset_size)).astype(int)
+
+        polycam_image_filenames = list(np.array(polycam_image_filenames)[idx])
+        # Copy images to output directory
+        copied_image_paths = copy_images_list(
+            polycam_image_filenames,
+            image_dir=image_dir,
+            crop_border_pixels=self.crop_border_pixels,
+            verbose=self.verbose,
+        )
+        num_frames = len(copied_image_paths)
+
+        copied_image_paths = [Path("images/" + copied_image_path.name) for copied_image_path in copied_image_paths]
+
+        if self.max_dataset_size > 0 and num_frames != num_images:
+            summary_log.append(f"Started with {num_frames} images out of {num_images} total")
+            summary_log.append(
+                "To change the size of the dataset add the argument --max_dataset_size to larger than the "
+                f"current value ({self.max_dataset_size}), or -1 to use all images."
+            )
+        else:
+            summary_log.append(f"Started with {num_frames} images")
+
```

Comment:
Unlikely to happen, but somewhere before calling polycam_to_json you should check `num_frames != 0` and `exit(1)` since polycam_to_json expects at least one frame.